src/main/java/uk/gov/companieshouse/pscstatementdataapi/api/PscStatementApiService.java

```diff
@@ -47,7 +47,7 @@ private ChangedResource mapChangedResource(String contextId, String companyNumbe
         event.setPublishedAt(String.valueOf(OffsetDateTime.now()));
         event.setType(DELETE_EVENT_TYPE);
         changedResource.setDeletedData(statement);
-        changedResource.setResourceUri(String.format(PSC_STATEMENTS_URI, companyNumber, statementId));
+        changedResource.setResourceUri(String.format(PSC_STATEMENTS_URI + "/internal", companyNumber, statementId));
```

Comment:
I think this should just be PSC_STATEMENTS_URI still, the resource uri will need to be used for the get request in stream.

See disqualified officers - https://github.com/companieshouse/disqualified-officers-data-api/blob/main/src/main/java/uk/gov/companieshouse/disqualifiedofficersdataapi/api/ResourceChangedRequestMapper.java#L22

Reply:
Okay, makes sense